Summers_James/app_base.htm

```diff
@@ -0,0 +1,150 @@
+<%inherit file="/homepage/templates/base.htm" />
+<%! from catalog import models as cmod %>
+
+<%block name="header">
+            <nav class="navbar navbar-inverse">
+                    <div class="navbar-header">
+                        <a class="navbar-brand" href="#" style="position: absolute; top:15px; left: 1px;">
+                          <img src="${ STATIC_URL }homepage/media/images/mylogo.png" alt="Site Logo" style="width:30px; height:30px;">
+                        </a>
+                      </div>
+                    <div class="container-fluid" style="font-size:20px;">
+                      <ul class="nav navbar-nav">
+                        <li class= " ${ 'active1' if request.dmp.page == 'index' else ''} " tabindex=1><a class="paddingright" href="/index">Home</a></li>
+                        <li class = " ${ 'active1' if request.dmp.page == 'about' else ''} " tabindex=2><a class="paddingright" href="/about">About</a></li>
+                        <li class = " ${ 'active1' if request.dmp.page == 'contact' else ''} " tabindex=3><a class="paddingright" href="/contact">Contact</a></li>
+                        <li class = " ${ 'active1' if request.dmp.page == 'terms' else ''} " tabindex=4><a class="paddingright" href= "/terms" >Terms</a></li>
+                        <li class = " ${ 'active1' if request.dmp.page == 'faq' else ''} " tabindex=5><a class="paddingright" href="/faq">FAQ</a></li>
+                        <li class = " ${ 'active1' if request.dmp.page == 'catalog' else ''} " tabindex=6><a class="paddingright" href="/catalog">Catalog</a></li>
```

Comment:
Since there isn't an `active2` anywhere (AFAIK), you could just change this to `active`

Reply:
It is a crazy story. I didn't want to use the bootstrap "active" so I made a custom "active1" I should just over ride it and delete the active in boostrap css